ibis/backends/clickhouse/tests/test_functions.py

```diff
@@ -532,3 +532,11 @@ def test_count_distinct_with_filter(alltypes):
     expected = alltypes.string_col.execute()
     expected = expected[expected.astype('int64') > 1].nunique()
     assert result == expected
+
+
+def test_group_concat(con, alltypes, translate):
+    t = alltypes
+    expr = t.string_col.group_concat(',')
+    expected = """arrayStringConcat(groupArray(`string_col`), ',')"""
+    assert translate(expr) == expected
+    assert len(con.execute(expr))
```

Comment:
Can we be more specific on what this needs to return? Also, I guess the `len` is unnecessary, if `len(...) != 0` I guess this will evaluate to true without the `len`. But would be better to test the value, or whatever possible, more than just saying that this should return something not empty, if possible.

Reply:
I have add test for `where` param in `group_concat` .
emmmm，i'm new in ibis and haven't found the right way to load test data and run ci locally，so i have no ideal to specify the return value of the group_concat expr 😇.